packages/inputs-tiny/src/textfield/index.jsx

```diff
@@ -0,0 +1,125 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { SIZES_KEYS } from '@/src/constants';
+import Meta from '@/src/textfields_meta';
+import styles from './index.module.scss';
+
+const TextField = ({
+  label,
+  error,
+  value = '',
+  onChange,
+  attachmentLeft,
+  onAttachmentLeftClick,
+  attachmentRight,
+  onAttachmentRightClick,
+  size = 'medium',
+  className,
+  disableBorder,
+  maxLength,
+  placeholder,
+  disabled,
+  type = 'text',
+  ...rest
+}) => {
+  const [inFocus, setFocus] = useState(false);
```

Comment:
let's think together if it's ok here to have state (because we wanted to build fully controlled components)